### PR TITLE
clone upload_server_cert.sh to work with files

### PR DESCRIPTION
**Pull request by iangcarroll**
Helpful when bulk uploading certificates.

### CODE REVIEW

**Comment by eranmes** [comment]
Post-holiday ping - let us know when you squash the commits together.

**Comment by iangcarroll** [comment]
:+1:

File: cpp/client/upload_cert_file.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+export PYTHONPATH=${PYTHONPATH}:../python
+if [[ $# -eq 0 ]] ; then
+    echo "USAGE: $0 [cert] [log pubkey] [precert?] (ct server address)"
+    exit 1
+fi
+FILE=$1 # file to upload
+PUBKEY=$2 # PEM encoded public key
+PRECERT=$3 # true or false
+#CT_SERVER='ct.googleapis.com/pilot'
```
Comment (eranmes) [inline cpp/client/upload_cert_file.sh]: Remove commented-out code

File: cpp/client/upload_cert_file.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+cp $FILE $TMP
+if upload
+then
+    echo Done
+    rm $TMP
+else
+    echo Try fixing the chain
+    TMP2=`mktemp /tmp/cert.XXXXXX`
+    ./fix-chain.py $TMP | tee $TMP2
+    CHAIN_FIXED=1
```
Comment (eranmes) [inline cpp/client/upload_cert_file.sh]: Rather than do this (and check further up), how about simply mv-ing $TMP2 to $TMP if the chain was fixed?

**Comment by BlueJa** [comment]
Thanks for everything and that password shit he an alright kid you know but yeah kept getting in my profile when I said not to and you guys do do alot

**Pull request closed without merging**